LAYOUT_STYLE_GUIDE.md

```diff
@@ -454,6 +454,22 @@ See the [lesson template](./templates/lesson-template.md) for a more easily copy
 Or, check out the [project template](./templates/project-template.md) for a more easily copyable project file.
 ~~~
 
+Typically you want to ensure the link text describes the purpose of the link or where the link will redirect a user, and can often be the title of a blog article or video.
+
```

Comment:
What do you think about also explicitly mentioning the no `this`/`here` rule? Since even if someone does end up writing a descriptive link, it's still subject to that.
Perhaps then also providing an example of descriptive link text containing `this` being rewritten to still be descriptive but without the `this` inclusion?

Reply:
What about:

Typically you want to ensure the link text describes the purpose of the link or where the link will redirect a user, and can often be the title of a blog article or video. You should also do your best to avoid including "this" and "here" in the link text to avoid our linter from flagging it as an error, even if the link text is descriptive. Often times "this" or "here" aren't necessary as part of the link text, and may cause some confusion despite a descriptive text ("Where's here??").

```markdown
// Sufficient, but could be tweaked further
Check out [this video on flex-grow from CoolYoutuber](...url)
Go look at our [installations guide here](...url)

// After a slight change
Check out this [video on flex-grow from CoolYoutuber](...url)
Go look at our [installations guide](...url)
```

Reply:
Looks perfect :ok_hand: